spec/features/admin/admin_logs_in_spec.rb

```diff
@@ -2,26 +2,41 @@
 
 feature 'Admin logs in' do
   before(:each) do
-    @admin = User.create(email: "admin@example.com",
+    admin = User.create(email: "admin@example.com",
                          password: "password",
                          full_name: "jeff admin",
                          display_name: "admin",
                          role: 1)
 
+    allow_any_instance_of(ApplicationController).to receive(:current_user) { admin }
     visit root_path
   end
 
   scenario 'and can access functionality to modify items' do
-    allow_any_instance_of(ApplicationController).to receive(:current_user) { @admin }
+    expect(page).to have_content("Dashboard")
+    expect(page).to have_content("Edit Items")
+    expect(page).to have_content("Order Dashboard")
+    expect(page).to have_content("Create Categories")
 
-    visit admin_items_path
-    assert page.has_content?("All Items")
+    click_on "Edit Items"
+
+    expect(current_path).to eq(admin_items_path)
+
+    expect(page).to have_content("All Items")
   end
 
   scenario 'and can access order dashboard' do
-    allow_any_instance_of(ApplicationController).to receive(:current_user) { @admin }
+    click_on "Order Dashboard"
+
+    expect(current_path).to eq(admin_orders_path)
+
+    expect(page).to have_content("Order Dashboard")
+  end
+
+  scenario 'and can access functionality to create categories' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.